src/utils/getComponentCssSelector.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+/**
+ * Adjusts the css selector for the component's css to increase specificity when needed
+ */
+export default function getComponentCssSelector(componentName: string, options: Object) {
+  if (options && options.specificityClass) {
+    return `.${options.specificityClass} .${componentName}`
```

Comment:
I'd like to see us support more than one specificityClass... can we support an array here? While we think one level of increased specificity is enough for our purposes now, we don't know for certain and other consumers of the framework may need it as well.